_pages/plugins/btrack/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Tissue Branch Tracker
-logo: /media/logos/btrack.png
+logo: /media/logos/btrack_logo.png
```

Comment:
Also, just FYI: any supporting images you need on a wiki page at `/plugins/btrack/index` can live in a corresponding directory e.g. `/media/plugins/btrack/some-image.jpg` and then you can refer to it in the Markdown file using the Liquid directive more succinctly `{% include img src="some-image" %}`—it automatically infers the same path prefix, and automatically checks for common image suffixes.

But this trick is not applicable to things in `icons` or `logos`, since those are handled specially—just for general-purpose supporting images for things like screenshots, diagrams, etc.

Reply:
I made these changes for the images showing the panels